tools/src/commands/IosNativeUnitTests.ts

```diff
@@ -0,0 +1,86 @@
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs-extra';
+import path from 'path';
+
+import * as Directories from '../Directories';
+import * as Packages from '../Packages';
+
+export async function iosNativeUnitTests() {
+  const packages = await Packages.getListOfPackagesAsync();
+  let packagesTested: string[] = [];
+  let errors: any[] = [];
+  for (const pkg of packages) {
+    if (
+      !pkg.isSupportedOnPlatform('ios') ||
+      !(await pkg.hasNativeTestsAsync('ios')) ||
+      !pkg.podspecName
```

Comment:
```suggestion
      !(await pkg.hasNativeTestsAsync('ios'))
```
I suggest moving all these checks inside this method - it would be much cleared from the called perspective 😉